.github/CHANGELOG.md

```diff
@@ -103,12 +103,13 @@
 
 - Typos addressed in templates documentation.
   [(#1094)](https://github.com/PennyLaneAI/pennylane/pull/1094)
+- Added `flaky` as dependency for running tests in documentation. [(#1113)](https://github.com/PennyLaneAI/pennylane/pull/1113)
 
 <h3>Contributors</h3>
 
 This release contains contributions from (in alphabetical order):
 
-Thomas Bromley, Kyle Godbey, Josh Izaac, Daniel Polatajko, Chase Roberts, Maria Schuld.
+Chase Roberts, Daniel Polatajko, Josh Izaac, Kyle Godbey, Maria Schuld, Pritish Sehzpaul, Thomas Bromley.
```

Comment:
@PritishSehzpaul we sort contributor names alphabetically by _last_ name, not first name! Could you revert the ordering here, and insert your name in the correct position.

Reply:
Gotcha. Thanks for letting me know. I will raise a commit